libretroshare/src/gxs/gxstokenqueue.h

```diff
@@ -23,19 +23,22 @@
 #define R_GXS_TOKEN_QUEUE_H
 
 #include "gxs/rsgenexchange.h"
+#include "retroshare/rsservicecontrol.h"
 #include "util/rsthreads.h"
 
 
 struct GxsTokenQueueItem
 {
+public:
 
 	GxsTokenQueueItem(const uint32_t token, const uint32_t req_type) :
-	    mToken(token), mReqType(req_type) {}
+	    mToken(token), mReqType(req_type), mStatus(RsTokenService::PENDING) {}
```

Comment:
I would have used RsTokenService::UNKNOWN, as a way to indicate that it's not initialized.